src/common/systems/GameController.hpp

```diff
@@ -52,8 +52,8 @@ class GameController {
     sf::Int32 m_elapsedTime = 0;
     sf::Int32 m_timeAccumulator = 0;
 
-    TemporalMetric m_gameStepMetric{120, sf::seconds(0.5f)};
-    TemporalMetric m_gameUpdateMetric{120, sf::seconds(0.5f)};
+    TemporalMetric m_gameStepMetric{30, sf::seconds(0.5f)};
+    TemporalMetric m_gameUpdateMetric{30, sf::seconds(0.5f)};
```

Comment:
It would be nice to make these constants. 
+ i want to make a todo that surfaces this number to the UI so there is more transparency of how the metric was calculated. Im not too worried about it being verbose since it's for debugging anyways